packages/tools/pluggable-widgets-tools/package.json

```diff
@@ -55,19 +55,21 @@
     "@types/testing-library__jest-dom": "^5.14.1",
     "@typescript-eslint/eslint-plugin": "^5.8.1",
     "@typescript-eslint/parser": "^5.8.1",
-    "@wdio/cli": "^7.5.3",
-    "@wdio/jasmine-framework": "^7.5.3",
-    "@wdio/local-runner": "^7.5.5",
-    "@wdio/selenium-standalone-service": "^7.5.3",
-    "@wdio/spec-reporter": "^7.5.3",
-    "@wdio/sync": "^7.5.3",
+    "@wdio/cli": "^7.16.13",
```

Comment:
do we need this `@wdio` deps?

Reply:
Yeap, we dont want to remove it for final customers.. we just removed sync as it doesnt work with node 16